### PR TITLE
Created new file for the town of Glen Ridge, NJ

### LINKED ISSUE #23: Add rparkin Hometown Tips
## Glen Ridge
- [ ] Create a Branch
- [ ] Add my File
- [ ] Create a pull
- [ ] Discuss
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by rparkin**
Extended description woud go here if you really want to type a longer description, closes #23

### CODE REVIEW

**Comment by cwarkulw** [comment]
Chaz left a comment about this pull request

File: glenRidge.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Glen Ridge has a great school system.  It is only one square mile.
```
Comment (githubteacher) [inline glenRidge.txt]: Is there any room for a restaurant?

**Comment by githubteacher** [comment]
Hey @githubteacher - can you take a look at this?

**Comment by rparkin** [comment]
Chaz, please approve.

**Comment by kishorebolinidi** [comment]
Hola.. :last_quarter_moon_with_face:

**Comment by cwarkulw** [comment]
I agree to merge based on the recommendation on the one and only restaurant in town.

**Pull request merged**